gbpservice/nfp/service_plugins/loadbalancer/drivers/nfp_lbaas_plugin_driver.py

```diff
@@ -0,0 +1,22 @@
+from gbpservice.nfp.config_orchestrator.agent import topics
+from gbpservice.nfp.configurator.drivers.loadbalancer.v1.haproxy import (
+    haproxy_lb_driver
+)
+from neutron_lbaas.services.loadbalancer.drivers.common import (
+    agent_driver_base as adb
+)
+
+
+class HaproxyOnVMPluginDriver(adb.AgentDriverBase):
+    device_driver = haproxy_lb_driver.DRIVER_NAME
+
+    def __init__(self, plugin):
+        # Monkey patch LB agent topic and LB agent type
+        adb.topics.LOADBALANCER_AGENT = topics.LB_NFP_CONFIGAGENT_TOPIC
+        adb.q_const.AGENT_TYPE_LOADBALANCER = 'NFP Loadbalancer agent'
+
+        super(HaproxyOnVMPluginDriver, self).__init__(plugin)
+
+        # TODO (RPM): RPC API version 2.0 is not integration tested
```

Comment:
Lets fix this in config agent during firewall integration testing and fix this in the plugin.